docs/config_samples/dataset_types/landsat8_example_product.yaml

```diff
@@ -0,0 +1,48 @@
+name: landsat8_example_product
+description: Landsat 8 example product
+metadata_type: eo3
+
+metadata:
+    product:
```

Comment:
`product.name` is already a unique enough constraint. So, I would say one would either have

```yaml
metadata:
  product:
    name: the_name
```
For fixed name products/datasets/

Or have structural matching via something like this:

```yaml
metadata:
  properties:
    eo:instrument: OLI_TIRS
    eo:platform: landsat-8
```

Reply:
Good point. Addresssed with new commit (9fd6d51). Hopefully the rst is about right - difficult to confirm with VSC previewer.